base_layer/wallet/src/wallet.rs

```diff
@@ -130,7 +130,7 @@ where
         let transaction_backend_handle = transaction_backend.clone();
 
         let factories = config.factories;
-        let (publisher, subscription_factory) = pubsub_connector(runtime.handle().clone(), 1500);
+        let (publisher, subscription_factory) = pubsub_connector(runtime.handle().clone(), 1_500);
```

Comment:
Same here. Maybe add it to config

Reply:
Hi, good comment, but in my opinion, these values should not be part of our normal `*.toml` config file, as it is really at a much too low level. 
I propose we rather create special command line parameters for these that can override (increase) the defaults, which we can then reduce back the way they were or close to the values before we started with the stress tests (100 vs. 1500 for the base node, 1000 vs. 50000 for the base node wallet, and 100 vs. 1500 for the mobile wallet) . This way it will enable us to start a base node with increased buffers when needed, and prevent attacks @sdbondi and @philipr-za mentioned where an attacker fills up your pubsub buffers with 8MB (equal to the msg frame size) messages.
I also propose we limit the command line option for buf_size to only the base node and base node wallet.
Opinions?

Reply:
Command line parameters and configs are basically equivalent IMHO. Power users who have more or less resources available may want to tweak these values, perhaps at their own risk. That's just my thoughts though

Reply:
Ah, never mind, config file it is :-).